### PR TITLE
Home page design converted to react

### PR DESCRIPTION
**Pull request by animeshgarai09**
Just converted from vanilla HTML CSS to react components.

components to check:-
- [src/components/footer/footer.js](https://github.com/animeshgarai09/Felix-Store-React/pull/2/files#diff-660865fb8b9eb7c700e0af02dc9cf2b1fcdf4a081e4bad527c91b4eab631f9f6)
- [src/components/header/header.js](https://github.com/animeshgarai09/Felix-Store-React/pull/2/files#diff-482aadf97cfd2f2ae994bc8f3c687f4e1178f3b3ecf79d2fd6e3196377c1580b)
- [src/pages/home/home.js](https://github.com/animeshgarai09/Felix-Store-React/pull/2/files#diff-1288430d7a8bef35cd045821519a80de3513c8ffb2196ce47dc0ff63b4b05a89)
- [src/pages/home/sub-components/data.js](https://github.com/animeshgarai09/Felix-Store-React/pull/2/files#diff-f55e2ebeafa1458a22cbce23eeb37f095c7e11ab8992da7f719c077e4590494e)

### CODE REVIEW

**Review by foolhardy21 — APPROVED** [review]
File structure looks good. react rewire-alias package is new to me, it looks helpful.

**Review by abhisheksahani-nova — APPROVED** [review]
Changes are good , folder structure is also very subtle, great work

File: src/pages/home/sub-components/categories.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { categoryList } from "./data"
+import styles from "../home.module.scss"
+import { Image } from "react-felix-ui"
+const Catagories = () => {
+    return (
+        categoryList.map((item, i) => {
+            return (
+                <a key={i} href="#" className={styles.item}>
```
Comment (Maianki) [inline src/pages/home/sub-components/categories.js:8]: Nitpicking: Better not to use the index as a key.
Here is a great article you can refer to why using the index as the key is anti-pattern.:https://robinpokorny.medium.com/index-as-a-key-is-an-anti-pattern-e0349aece318

File: src/pages/home/sub-components/features.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { featureList } from "./data"
+import styles from "../home.module.scss"
+
+const Features = () => {
+    return (
+        featureList.map((item, i) => {
+            return (
+                <div key={i} className={styles.feature}>
```
Comment (Maianki) [inline src/pages/home/sub-components/features.js:8]: Here too you can avoid using the index as a key.

**Review by Maianki — APPROVED** [review]
Great job learned a few things from your code. Looks good to me.

File: src/pages/home/sub-components/categories.js (hunk 1 above)
Comment (animeshgarai09) [reply src/pages/home/sub-components/categories.js:8]: Using index just for now. will change after I connect it with the backend.

**Pull request merged**